panels/user-accounts/data/app-permissions.ui

```diff
@@ -126,7 +130,9 @@
           <object class="GtkLabel" id="system_installation_label">
             <property name="visible">True</property>
             <property name="xalign">1.0</property>
-            <property name="label" translatable="yes">Installed apps can be used by every user:</property>
+            <property name="label" translatable="yes">Install Apps for All _Users:</property>
```

Comment:
The argument for changing this string is basically that, when translated into at least French and Portuguese, it forces the window to expand horizontally:

![screenshot from 2019-01-18 10-23-56](https://user-images.githubusercontent.com/86760/51383298-f8da6600-1b10-11e9-9aed-ba4b6c055a6f.png)
![captura de tela de 2019-01-18 10-45-09](https://user-images.githubusercontent.com/86760/51383300-fb3cc000-1b10-11e9-93b8-00a9a5a7c68e.png)

Reply:
The other argument for changing it is that it isn’t in Title Case and can’t be mnemonic-ised properly in its previous state.